### PR TITLE
Transfer content from Create Your Pipeline to Programming Guide.

### PR DESCRIPTION
**Pull request by hadarhg**
Transfer content from Create Your Pipeline to Programming Guide. 
Delete Create Your Pipeline and remove from top menu.

### CODE REVIEW

**Comment by hadarhg** [comment]
R: @francesperry

**Comment by hadarhg** [comment]
R: @davorbonaci

**Comment by hadarhg** [comment]
There's a problem with the Python code not showing up for snippets in Configuring Pipeline Options section.

File: src/documentation/pipelines/create-your-pipeline.md [hunk 1]
```diff
@@ -1,161 +0,0 @@
----
```
Comment (davorbonaci) [inline src/documentation/pipelines/create-your-pipeline.md]: Should we leave a redirect to the new location?

**Pull request closed without merging**